recipes/snpeff/5.0/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set snpeff_ver = "v5_0e" %}
+# NOTE: if the version contains a trailing letter, use the <d>.<d>.1<l> format
+# Please update recipes/snpsift as well!
+{% set version = "5.0" %}
+{% set sha256 = "85d907b5dd9e9008a0cf245956e3c9077a31e45f21a1b580d9b98a53fd8dcb9d" %}
+
+about:
+  home: 'http://snpeff.sourceforge.net/'
+  license: "LGPLv3"
+  summary: "Genetic variant annotation and effect prediction toolbox"
+
+package:
+  name: snpeff
+  version: {{ version }}
+
+build:
+  number: 2
+  noarch: generic
+  run_exports:
+    - {{ pin_compatible('snpeff', max_pin="x.x") }}
```

Comment:
I would appreciate if someone who knows SnpEff well could make sure it makes sense to have `- {{ pin_compatible('snpeff', max_pin="x.x") }}` here. It was not in 5.0's original recipe.